src/scripts/main.js

```diff
@@ -1 +1,6 @@
 'use strict';
+
+document.querySelector('#landing-form')
+  .addEventListener('click', function(event) {
+    event.preventDefault();
+  }, false);
```

Comment:
umm, it's not quite that.
'click' event for the form is not the right event - for the form it would be 'submit' - you don't have to click to submit, right? You could just hit `enter` on keyboard.
And you might notice that still form fields validation doesn't drop validation helpers.

```
const form = document.querySelector('<form selector>');

form.addEventListener('submit', function(event) {
  event.preventDefault();
  form.reset();
});
```

Reply:
Okay, makes sense. Fixed.